filebeat/filebeat.full.yml

```diff
@@ -246,11 +246,11 @@ filebeat.prospectors:
 # sub-dictionary. Default is false.
 #fields_under_root: false
 
-# How often (in seconds) shippers are publishing their IPs to the topology map.
+# DEPRECATED: How often (in seconds) shippers are publishing their IPs to the topology map.
```

Comment:
I didn't put other deprecated options in the full configs. I think we can remove them completely.